pipelines/concourse.yml

```diff
@@ -1838,6 +1859,16 @@ resources:
     client_secret: ((bosh_client.secret))
     deployment: concourse-prod
 
+- name: bosh-topgun-worker-deployment
```

Comment:
I also think this resource should have `tags: [topgun]` https://concourse-ci.org/resources.html#resource-tags